de-lib/src/main/java/org/iplantc/de/admin/desktop/client/permIdRequest/views/MetadataDialog.java

```diff
@@ -34,16 +31,17 @@ public void onSelect(SelectEvent event) {
     private class OkSelectHandler implements SelectEvent.SelectHandler {
         @Override
         public void onSelect(SelectEvent event) {
-            if (!metadataView.isValid()) {
-                ConfirmMessageBox cmb = new ConfirmMessageBox(I18N.DISPLAY.error(),
-                                                              appearance.metadataSaveError());
+            if (!meta_pre.isValid()) {
+                ConfirmMessageBox cmb =
+                        new ConfirmMessageBox(I18N.DISPLAY.error(), appearance.metadataSaveError());
```

Comment:
Minor thing, but I've been trying to get rid of these I18N references and using appearance classes  instead.